src/ChainManagerRegistry.php

```diff
@@ -31,7 +32,7 @@ public function getRepository($persistentObject, $persistentManagerName = null):
         foreach ($this->managerRegistries as $managerRegistry) {
             try {
                 return $managerRegistry->getRepository($persistentObject, $persistentManagerName);
-            } catch (MappingException) {
+            } catch (MappingException|ORMMappingException) {
```

Comment:
not catching this exception throws an error in `MakeFactoryTest::can_create_factory_with_default_enum()` with data set `odm`

not sure why we've not seen this before :thinking: